test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsModelValidatorTest.cs

```diff
@@ -64,15 +58,15 @@ public static IEnumerable<object[]> Validate_SetsMemberName_OnValidationContext_
             string expectedMemberName)
         {
             // Arrange
-            var attribute = new Mock<ValidationAttribute> { CallBase = true };
-            attribute.Protected()
-                     .Setup<ValidationResult>("IsValid", ItExpr.IsAny<object>(), ItExpr.IsAny<ValidationContext>())
-                     .Callback((object o, ValidationContext context) =>
-                     {
-                         Assert.Equal(expectedMemberName, context.MemberName);
-                     })
-                     .Returns(ValidationResult.Success)
-                     .Verifiable();
+            var attribute = new Mock<TestableValidationAttribute> { CallBase = true };
+            attribute
+                .Setup(p => p.IsValidPublic(It.IsAny<object>(), It.IsAny<ValidationContext>()))
+                .Callback((object o, ValidationContext context) =>
+                {
+                    Assert.Equal(expectedMemberName, context.MemberName);
+                })
+                .Returns(ValidationResult.Success)
+                .Verifiable();
```

Comment:
It does - there was a bug when I initially ported it which apparently was later fixed. Haven't seen if it works since the fix